l10n_it_mis_reports/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Italy MIS Builder templates',
```

Comment:
Espliciterei i report generati, in italiano. Tipo "ITA - MIS builder - Bilancio civilistico"

Nel nome del modulo espliciterei, tipo ` l10n_it_mis_reports_pl_bs`.

Anche il commit dovrebbe riportare almeno il nome del modulo

Reply:
non ho specificato pl_bs perchè potrebbero esserci anche altri tipi di report da includere tipo MARGINE OPERATIVO LORDO (A+B+C) = EBITDA ecc.

Reply:
Teoricamente "l10n_it_mis_reports" identifica qualunque "Management Information System" report legato alla localizzazione italiana. Se teniamo quindi questo nome, questo modulo dovrà contenere in futuro qualunque report "MIS". Non è detto che li si voglia installare sempre tutti in un colpo.
Secondo me è più chiaro a agevole avere moduli più specifici, quindi se prevedi di fare i template per EBITDA, farei un modulo dedicato.
Se ci sono parti comuni a tutti i moduli dei report "MIS", si può fare il modulo `l10n_it_mis_reports_base` come dipendenza nascosta.

Reply:
ok